stats/stats-proto/build.rs

```diff
@@ -16,33 +16,55 @@ fn compile(
         .protoc_arg("--openapiv2_opt")
         .protoc_arg("grpc_api_configuration=proto/api_config_http.yaml,output_format=yaml,allow_merge=true,merge_file_name=stats,json_names_for_fields=false")
         .bytes(["."])
-        .type_attribute(".", "#[actix_prost_macros::serde(rename_all=\"snake_case\")]")
+        .type_attribute(".blockscout.stats", "#[actix_prost_macros::serde(rename_all=\"snake_case\")]")
         .field_attribute(
             ".blockscout.stats.v1.HealthCheckRequest.service",
             "#[serde(default)]"
         )
         .field_attribute(".blockscout.stats.v1.Point.is_approximate", "#[serde(skip_serializing_if = \"std::ops::Not::not\")]")
         .field_attribute(".blockscout.stats.v1.Point.is_approximate", "#[serde(default)]")
         .field_attribute(".blockscout.stats.v1.GetLineChartRequest.resolution", "#[serde(default)]");
+    blockscout_endpoint_health::add_to_compile_config(&mut config);
 
     config.compile_protos(protos, includes)?;
     Ok(())
 }
 
+fn vec_path_buf_to_string(v: Vec<PathBuf>) -> Vec<String> {
+    v.into_iter()
+        .map(|path| {
+            path.to_str()
+                .expect("Non UTF-8 paths are not supported")
+                .to_string()
+        })
+        .collect()
+}
+
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let swagger_crate_folder = Path::new("../../libs/endpoints/health");
+
     // We need to rebuild proto lib only if any of proto definitions
     // (or corresponding http mapping) has been changed.
-    println!("cargo:rerun-if-changed=proto/");
+    let mut proto_files_folders = ["proto/"].map(PathBuf::from).to_vec();
+    proto_files_folders.extend(blockscout_endpoint_health::includes(swagger_crate_folder));
+    let proto_files_folders = vec_path_buf_to_string(proto_files_folders);
+
+    for folder in &proto_files_folders {
+        println!("cargo:rerun-if-changed={folder}/");
+    }
+
+    let mut protos = ["proto/stats.proto"].map(PathBuf::from).to_vec();
+    protos.extend(blockscout_endpoint_health::proto_files(
+        swagger_crate_folder,
+    ));
+    let protos = vec_path_buf_to_string(protos);
```

Comment:
I wonder if the main() does too many things. Should we may be isolate path setup in a separate function?